### DISCUSSION TITLE
Why serverless command usage must depend on serverless-tencent?

### DISCUSSION THREAD

**Discussion by SquatsTonight** [serverless/serverless#11172] [Q&A]
### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Use case description

![image](https://user-images.githubusercontent.com/52195264/174935159-632cc175-c25f-4025-990b-f4ce4205ce53.png)

**Comment by pgrzesik** [comment]
Hello @git-qfzhang - could you share a bit more about your setup? Are you using Components and are located in China? If that's the case, by default the triage process redirects to use `serverles-tencent`.
